From ea73b069dd83001a90c47fea1222206015bcbbea Mon Sep 17 00:00:00 2001
From: Leo Singer <leo.singer@ligo.org>
Date: Wed, 5 Aug 2020 11:19:26 -0400
Subject: [PATCH] Remove unnecessary install dependencies on setuptools, wheel

These packages are needed for running the build system, but they
are not required at run time. They should not be listed in
`install_requires`.
---
 setup.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)
 mode change 100644 => 100755 setup.py

diff --git a/setup.py b/setup.py
old mode 100644
new mode 100755
index 1d2b436..cbcb55d
--- a/setup.py
+++ b/setup.py
@@ -24,10 +24,7 @@
     "Programming Language :: Python :: 3",
 ]
 INSTALL_REQUIRES = [
-    "matplotlib>=2.1",
-    "setuptools>=40.6.0",
-    "setuptools_scm",
-    "wheel",
+    "matplotlib>=2.1"
 ]
 EXTRA_REQUIRE = {
     "test": [
